services/director-v2/src/simcore_service_director_v2/utils/async_utils.py

```diff
@@ -60,13 +61,23 @@ def get_context(args, kwargs: Dict) -> Context:
 
             key_parts = deque()
             for arg in target_args:
-                if arg not in search_args:
+                sub_args = arg.split(".")
```

Comment:
could you please update the docstring of this function to reflect the new changes.

Reply:
done